content/php-client/resources/family-variants.md

```diff
@@ -30,3 +30,34 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
  */
 $familyVariant = $client->getFamilyVariantApi()->get('boots', 'boots_color_size');
 ```
+
+### Create a family variant
+
+If the family variant does not exist yet, this method creates it, otherwise it throws an exception.
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$client->getFamilyVariantApi()->create('boots', [
+    'code' => 'boots_size_color',
```

Comment:
`code` should be added as an argument of the `create` method like the other methods